content/en/docs/installing-on-kubernetes/on-prem-kubernetes/installing-kubesphere-on-minikube.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Installing KubeSphere on Minikube"
+keywords: 'KubeSphere, Minikube, Minimal, Installation'
+description: 'Install KubeSphere on an existing Minikube cluster with a minimal installation package.'
+linkTitle: "Installing KubeSphere on Minikube"
+weight: 2200,
+showSubscribe: true
+---
+
+In addition to installing KubeSphere on a Linux machine, you can also deploy it on minikube cluster. This tutorial demonstrates the general steps of completing a minimal KubeSphere installation on Minikube.
+
+## Prerequisites
+
+- To install KubeSphere 3.2.1 on Minikube, your Minikube version must be v1.23.x, v1.24.x, v1.25.x.
```

Comment:
KubeSphere v3.3.0 is available now. If this document applies to KubeSphere v3.3.0, we should change "KubeSphere 3.2.1" to "KubeSphere v3.3.0".

BTW, the master branch of this repository contains documents of KubeSphere v3.3.0. If this document applies only to KubeSphere v3.2.1, it should be sent to the release-3.2.1 branch of this repository.

Reply:
Okay, thanks for the review. This document is meant for Kubesphere'ss latest version. I'll update it after testing it on v3.3.0.

Reply:
Hi, I realized this document contains a video demonstration for Kubesphere 3.2.1, So, should I open up a new pull request for sending this document to release-3.2 branch? 
Or is it possible to change it from github only(like I see there is an option to change the base but it gives the following warning "Some commits from the old base branch may be removed from the timeline, and old review comments may become outdated." ) 
Which way would be the safest?

Reply:
On your local machine, you can create a branch synchronized with release-3.2, cherry-pick your commit to the branch, and then create a new pull request.

Reply:
I have opened up a new pull request. https://github.com/kubesphere/website/pull/2554